src/libsync/vfs/cfapi/vfs_cfapi.h

```diff
@@ -54,6 +54,9 @@ class VfsCfApi : public Vfs
     AvailabilityResult availability(const QString &folderPath) override;
 
     void cancelHydration(const QString &requestId, const QString &path);
+    bool hydrationCanceled(const QString &requestId);
+
+    void doneWithHydration(const QString &requestId);
```

Comment:
@FlexW How about `finishHydration`?

Reply:
That was the original name ;) But yes, why not. It's a little bit confusing because there is already this `finishedHydration` signal.